opennms-services/src/main/java/org/opennms/netmgt/collectd/Collectd.java

```diff
@@ -1489,7 +1493,7 @@ private void instantiateCollectors() {
                 }
             });
             if(!collectorFuture.isDone()) {
-                LOG.warn("The collector {} with class {} is not available.", svcName, collector.getClassName());
+                LOG.warn("The collector {} with class {} is not available yet, if the feature is installed correctly it will be available latter.", svcName, collector.getClassName());
```

Comment:
```suggestion
                LOG.warn("The collector {} with class {} is not available yet, if the feature is installed correctly it will be available later.", svcName, collector.getClassName());
```